qa/restart-upgrade/src/test/java/org/opensearch/knn/bwc/ModelIT.java

```diff
@@ -139,22 +136,6 @@ public void testKNNModelDefault() throws Exception {
         }
     }
 
-    // KNN Delete Model test for model in Training State
-    public void testDeleteTrainingModel() throws Exception {
```

Comment:
This logic no longer works because with @ryanbogan change, we mark models stuck in training state as failed. So, I deleted it. Also, we should avoid directly accessing the system index in future tests.